db-commons/src/main/resources/reference.conf

```diff
@@ -37,6 +37,11 @@ bitcoin-s {
         disable-console = false
     }
 
+    node {
+        spv-mode = yes # yes, no, true, false
+        neutrino-mode = yes # yes, no, true, false
+    }
```

Comment:
is there a reason this is specified here and also in `app/server/src/main/resources/application.conf`? Is there any possibility that these won't be in sync?

Reply:
`reference.conf` contains all the default values for our application. `application.conf` and `$HOME/.bitcoin-s/bitcoin-s.conf` is where the user can override the default values. So there's no reason to have the same values here as the default values in `reference.conf`